lib/ledger.js

```diff
@@ -99,147 +97,278 @@ async function _findAgent() {
 }
 
 // setup the genesis node
-function _setupGenesisNode(callback) {
-  async.auto({
-    ledgerOwner: callback => _getLedgerOwner({nodeType: 'genesis'}, callback),
+async function _setupGenesisNode() {
+  try {
+    const {FlexDocStore} = require('flex-docstore');
+    const didStore = FlexDocStore.using('files', {
```

Comment:
If this is not in the config directory (or a subdirectory under it), it'll be wiped if machines are ever reset via docker-compose rebuild... just something to keep in mind... anything that's not in the config directory or the mongo/redis database can be blown away.

Reply:
Added exfiltrating these documents to the roadmap.